### ISSUE TITLE
designsystem.digital.gov accessibility testing

### ISSUE THREAD

**Issue by jAigret-Bix** [uswds/uswds#3733]
Follow up to https://github.com/uswds/uswds-team/issues/7
https://github.com/uswds/uswds-site/blob/aad70f3749f75eb9a2a310e1bc422470d61f26f9/config/run-axe.js#L12

- [x] expand number of pages tested w CI w each build of site
- [x] Define pages to test. Use good selection of common pages. 
- [x] new issues created from A11y violations 

DOD
- [ ] all related A11y violation issues are complete.  [#1093](https://github.com/uswds/uswds-site/issues/1093) and #1094 

BLOCKED:  the issues are blocked until other streams on Component page updates are done

uswds/uswds-site#1109
uswds/uswds-site#1110
uswds/uswds-site#1111
uswds/uswds-site#1108

**Comment by mejiaj** [comment]
@jAigret-Bix the issues are blocked until other streams on Component page updates are done

https://github.com/uswds/uswds-site/issues/1109
https://github.com/uswds/uswds-site/issues/1110
https://github.com/uswds/uswds-site/issues/1111
https://github.com/uswds/uswds-site/issues/1108

**Comment by thisisdano** [comment]
Each of these accessibility items has its own issue. Closing this one.
